linera-rpc/tests/snapshots/format__format.yaml.snap

```diff
@@ -114,7 +114,8 @@ BlobContent:
   STRUCT:
     - blob_type:
         TYPENAME: BlobType
-    - bytes: BYTES
+    - bytes:
+        SEQ: U8
```

Comment:
`BYTES` is better

Reply:
Yes, but how to do it?
This type is written by `serde-reflection::Tracer`, and the configuration of the tracer does not appear to leave room for choosing `BYTES`.

Reply:
Thank you for pointing this out.

However, the `serde_as` works for `Arc<Box<[u8]>>` but not for `Arc<[u8]>`.

Reply:
Ok, found a way to do that.